### PR TITLE
Built out resources component based on Student Figma Deisgn flow

### PR DESCRIPTION
**Pull request by leewp8**
Co-authored-by: Spencer Longhurst spencersean12@gmail.com
Co-authored-by: Ahmad Wahidy shah.wahidy@gmail.com
Co-authored-by: Jose Mercado josefmercado99@gmail.com

## Description

Created a Resources component for the student flow based on the Figma design. Added a route, banner, and sidebar functionality to correspond with the component page.

Fixes # (issue)

## Loom Video

https://www.loom.com/share/a6b60db045194045977919426e581443

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by kylerostler — DISMISSED** [review]
Looks good could use more styling overall but the skeleton is there. Set up nicely so that it can be connected to redux and styled in the future. Clean code no unecessary comments or console.logs.

**Review by Ogden-R — APPROVED** [review]
i love that you were able to make cards and components for those. these components look very nice and now just need a little bit of love connecting them to the instructors stuff. great job!

**Review by ashtilawat23 — CHANGES_REQUESTED** [review]
Looks like you deleted the entire package-lock. We don't want to delete it but preserve the one on Main. I would open your branch in VScode. Open the raw text from the package-lock in the browser. Use Cmd A to select all, Cmd C to copy, and Cmd V to paste into your branch. If you are on PC, it's Control.

**Review by brandib222 — APPROVED** [review]
Great start on the student resources page! Good use of antd to keep things simple and make things easier to style/ make dynamic in the future.

File: src/components/pages/StudentHome/StudentResources.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+const StudentResourcesPage = () => {
+  const { Content } = Layout;
+  return (
+    <Layout>
+      <StudentSidebar active="Resources" />
+      <Content className="resources-container">
+        <Banner />
+        <div className="cardContainer">
+          <Card className="resourceCard">
```
Comment (Michael-Subbarao) [inline src/components/pages/StudentHome/StudentResources.js:16]: I would suggest separating this card out into it's own component for reuse.

**Review by Michael-Subbarao — APPROVED** [review]
Overall it looks like a great start to the component and everything looks well designed. Code looks clean as well. I might make the StudentResources component slightly more modular but that can be done when it gets hooked up to the backend.

**Review by anastasia-lapteva — APPROVED** [review]
Hello William, Spencer, Ahmad, and Jose,

Thank you for thoroughly walking us through all relevant information from the Jira Kanban board, to the Figma design, to the code, to your local deployment. I find that getting started drawing something on the blank screen is often the most difficult part...good job with this!

**Comment by laurensruiz** [comment]
The basic structure of the page looks great and might need more styling but the overall code looks clean.

**Review by wreck888 — DISMISSED** [review]
I would recommend not hardcoding the data but instead making seed files for them in the backend. Other than that the code looks great!

**Review by EmmS21 — APPROVED** [review]
Great start building out the component, code looks pretty clean

**Review by laurensruiz — DISMISSED** [review]
The basic structure of the page looks great and might need more styling. Antd design components is a bit hard to style at first, but once you get a good grasp of it, it'll be easier. Great work!

**Pull request merged**